src/AppEntry.js

```diff
@@ -42,12 +44,12 @@ const AppEntry = ({ useLogger, connectToDevTools }) => {
     registry.register({ notifications });
 
     useEffect(() => {
-        insights.chrome.init();
-        insights.chrome.identifyApp('malware');
-        if (insights.chrome?.globalFilterScope) {
-            insights.chrome.on('GLOBAL_FILTER_UPDATE', ({ data }) => {
+        chrome.init();
+        chrome.identifyApp('malware');
+        if (chrome?.globalFilterScope) {
```

Comment:
It should be ok now to remove the optional chaining with the `?` from `chrome`. 
It seems to have been redundant before as well. I suspect this is to please tests when there is no `insights` set in the global scope.

Reply:
Ah seems youre right. I've removed them in the new commit. Thanks for taking a look!